docs/UserGuide.md

```diff
@@ -241,15 +273,34 @@ This also means that you can use `t/` to remove all tags from a dentist using th
 
 ## Part 2 - Patient Features
 
-### Adding a patient: `add-patient`
-
+### Adding a patient : `add-patient`
```

Comment:
Are we standardising to have a space before the colon? If so, do change for the dentist features above :)